src/main/java/seedu/address/ui/MainWindow.java

```diff
@@ -156,7 +156,7 @@ void fillInnerParts() {
 
         // Must add a getFilteredProcedureList in logic for a single client-- works with the list method
         List<Procedure> procList = new ArrayList<>();
-        procList.add(new Procedure(new Information("info"), new Date("14/03/2022"), new Cost("30")));
+        // procList.add(new Procedure(new Information("info"), new Date("14/03/2022"), new Cost("30")));
```

Comment:
Maybe this could be deleted since this is redundant code and easily re-writeable :).